homeassistant/components/spotify/media_player.py

```diff
@@ -161,6 +160,18 @@ def media_content_id(self) -> str | None:
         item = self._currently_playing.get("item") or {}
         return item.get("uri")
 
+    @property
+    def media_content_type(self) -> str | None:
+        """Return the media type."""
+        if not self._currently_playing:
+            return None
+        item = self._currently_playing.get("item") or {}
+        return (
+            MediaType.PODCAST
+            if item.get("type") == MediaType.EPISODE
+            else MediaType.MUSIC
```

Comment:
Please don't use multi-line ternary statements

Reply:
Think that may have been an auto-fix from one of the linters. Easily resolved, though!